### PR TITLE
chore: bump stacks core to `signer-3.0.0.0.1`

### PR DESCRIPTION
**Pull request by cylewitruk**
## Description

Closes: #774 

## Changes

Updates `Crates.toml`, some test fixture files and the `devenv` to use commit `b26f406fc0bfd271a5cd5b54ccb064e7d3a0650a` which is the latest non-RC release as of today.

There were several breaking configuration changes and a new field in the `v2/info` response which needed to be adjusted, but everything looks OK running in devenv now.

## Checklist:

- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: signers.sh [hunk 1]
```diff
@@ -40,6 +40,9 @@ exec_run() {
   docker compose -f "$DOCKER_COMPOSE_PATH" --profile sbtc-postgres down
   docker compose -f "$DOCKER_COMPOSE_PATH" --profile sbtc-postgres up --detach
 
+  # Wait for the postgres instances to start (can get ssl handshake errors etc. otherwise)
+  sleep 1
```
Comment (cylewitruk) [inline signers.sh:44]: this snuck in but just as well to get it merged, i ran into problems without it

File: docker/stacks/stacks-regtest-miner.toml [hunk 2]
```diff
@@ -43,28 +59,23 @@ auth_token = "12345"
 # Add stacks-api as an event observer
 [[events_observer]]
 endpoint = "stacks-api:3700"
-retry_count = 255
-include_data_events = false
```
Comment (matteojug) [inline docker/stacks/stacks-regtest-miner.toml:47]: qq: were those deprecated?

**Review by matteojug — APPROVED** [review]

File: docker/stacks/stacks-regtest-miner.toml (hunk 2 above)
Comment (cylewitruk) [reply docker/stacks/stacks-regtest-miner.toml:47]: not even deprecated, hard config error now

**Comment by cylewitruk** [comment]
Suggest we wait with merging this until after the demo to not introduce any potential new unknowns

**Pull request merged**